packages/thirdweb/src/exports/extensions/marketplace.ts

```diff
@@ -31,6 +31,16 @@ export {
   type CurrencyPriceForListingParams,
 } from "../../extensions/marketplace/direct-listings/read/currencyPriceForListing.js";
 
+// isDirectListingSupported
+export { isCreateListingSupported } from "../../extensions/marketplace/__generated__/IDirectListings/write/createListing.js";
+export { isCancelListingSupported } from "../../extensions/marketplace/__generated__/IDirectListings/write/cancelListing.js";
+export { isBuyFromListingSupported } from "../../extensions/marketplace/__generated__/IDirectListings/write/buyFromListing.js";
+
+// isEnglishAuctionSupported
+export { isCreateAuctionSupported } from "../../extensions/marketplace/__generated__/IEnglishAuctions/write/createAuction.js";
+export { isBidInAuctionSupported } from "../../extensions/marketplace/__generated__/IEnglishAuctions/write/bidInAuction.js";
+export { isCancelAuctionSupported } from "../../extensions/marketplace/__generated__/IEnglishAuctions/write/cancelAuction.js";
```

Comment:
this likely is a little more involved, marketplace is an (old style) extensions contract so we have to detect that first -> then ask it for its extensions -> then check if one of the extensions supports the bytecode, have to think about this a bit more

Reply:
I'm afraid I don't get the idea here. Will fix the exports first

Reply:
> marketplace is an (old style) extensions contract so we have to detect that first

To be specific I don't get this one :D

Reply:
yeah this will work fine